Src/libCZI/CziDisplaySettings.h

```diff
@@ -15,8 +15,8 @@ class CChannelDisplaySettingsOnPod : public libCZI::IChannelDisplaySetting
 private:
     libCZI::ChannelDisplaySettingsPOD cdsPod;
 public:
-    explicit CChannelDisplaySettingsOnPod(const libCZI::ChannelDisplaySettingsPOD& pod)
-        : cdsPod(pod)
+    explicit CChannelDisplaySettingsOnPod(libCZI::ChannelDisplaySettingsPOD pod)
```

Comment:
I've seen the change from passing const ref to pass by value a few times. Is there any large advantage or just mostly for style? 

Previously we would bind const ref to param and then copy into base object via constructor. Now we copy struct and then move into base. Wouldn't ideal form be passing via const ref and then moving into base?